apps/site/assets/css/_transit-near-me.scss

```diff
@@ -230,6 +230,12 @@ $xxl-map-height: map-get($container-max-widths, xxl) * math.div(9, 16);
     }
   }
 
+  .c-link-block__inner-link {
```

Comment:
```suggestion
  &.u-bg--bus  .c-link-block__inner-link {
```

This was affecting all modes, not only bus - so it moved the icon for subway and commuter rail alerts too. :(

<img width="193" alt="image" src="https://user-images.githubusercontent.com/2136286/216656274-53419b05-bccb-4160-84c4-af9d094e21a8.png">

But this change should fix that.

Reply:
I had to test locally since there weren't enough subway/rail alerts on dev-blue or dev-green 👀